Assets/MixedRealityToolkit/Inspectors/Setup/ProjectPreferencesInspector.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEditor;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Editor
+{
+    [CustomEditor(typeof(ProjectPreferences))]
+    public class ProjectPreferencesInspector : UnityEditor.Editor
```

Comment:
Technically this is public class, so it should have comment...though perhaps we should make exceptions for inspectors. I would prefer though to just always have principle "if it is public class or method, it should have doc so docfx shows something, even a simple message"

Reply:
Comments for inspectors will just get a little ridiculous because they will mostly all be "editor inspector for class blah" Only a handful of inspectors do dynamic options or updates via the inspector.

@julenka, possible alternative, assuming this works, make inspector classes internal? So it's not a public interface we have to cover/surface

Reply:
> @julenka, possible alternative, assuming this works, make inspector classes internal? So it's not a public interface we have to cover/surface

That sounds good to me.